src/core/File.ts

```diff
@@ -221,6 +221,7 @@ export class File {
      * @memberOf File
      */
     public tryPlugins(_ast?: any) {
+        if (this.context.runAllMatchedPlugins) { this.tryAllPlugins(_ast) }
```

Comment:
You're right, there was meant to be a return there. I've pushed a change to fix that.